src/components/GlobalSearch.jsx

```diff
@@ -4,12 +4,8 @@ import Button from './Button';
 function GlobalSearch({searchString}) {
     return (
           <form className="flex">  
-          <div className="w-full">
```

Comment:
u removed `w-full` but you didnt add it to input.
Did u test the changes ?

Reply:
yes i tested